app/components/CommentForm/CommentForm.css

```diff
@@ -70,6 +70,6 @@
 }
 
 .submit {
-  max-width: 150px;
-  margin-top: 30px;
+  /* max-width: 150px; */
```

Comment:
Any specific reason to comment these out instead of removing them?

Reply:
oops. It is removed in [this PR](https://github.com/webkom/lego-webapp/pull/4018) so I just wanted to quickly test that I didn't break anything without creating any conflicts. Forgot to uncomment though - thanks!